src/commands/lambda/README.md

```diff
@@ -21,16 +21,19 @@ Run `datadog-ci lambda instrument` to apply Datadog instrumentation to a Lambda.
 
 ```bash
 # Instrument multiple functions specified by names
-datadog-ci lambda instrument -f <function-name> -f <another-function-name> -r us-east-1 -v 46 -e 10
+datadog-ci lambda instrument -f <function-name> -f <another-function-name> -r us-east-1 -v 81 -e 49
```

Comment:
nit: I wonder if it would be possible to use the environment variables used in other repos?
https://github.com/DataDog/documentation/blob/f7c7b9848ef8fe726dd571c0c724142e4af05205/content/en/serverless/installation/nodejs.md?plain=1#L82

Reply:
I'm also interested in learning how the Documents repo works with those variables?

Reply:
Interesting, did a quick look and they're maintained manually in this file https://github.com/DataDog/documentation/blame/ef35033f170f6dab5537e9d08217dc67745d1765/layouts/shortcodes/latest-lambda-layer-version.html, I wonder if there is a way we can read from those here

Reply:
FTI: #websites is a good place to ask if you want to learn more about these